dev-test/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta charset="utf-8" />
-
+  <link href="/config.nrk.yml" type="text/yaml" rel="cms-config-url">
```

Comment:
This should be removed

Reply:
Yep, removed.

This was originally added as a workaround to create a development environment for me to add my own Azure DevOps settings. What's the expected workflow for developing backends? I see a series of folders in dev-test with the name of each backend with example YAML config, I presume I should create one for Azure too but I'm unclear how they're intended to be _used_.

Is the idea that they are templates, and during development we should copy whichever one to the root of dev-test, customise it, but then revert it before every commit?

Reply:
They are actually used for the e2e tests as templates for the tests. I'll add one for azure once I add the tests.
I use them as templates when doing manually testing too, and just don't commit them (like you described) by doing `git add packages/`.